zyte_parsers/price.py

```diff
@@ -0,0 +1,18 @@
+from decimal import Decimal
+from typing import Optional
+
+from price_parser import Price
+
+from zyte_parsers import SelectorOrElement
+from zyte_parsers.utils import extract_text
+
+
+def extract_price(node: SelectorOrElement) -> Optional[Decimal]:
+    """Extract a price value from a node that contains it.
+
+    :param node: Node including the price text.
+    :return: The price value or None.
+    """
+    text = extract_text(node)
+    price_parsed = Price.fromstring(text)
+    return price_parsed.amount
```

Comment:
What you you think about just returning the Price object here? Currently some data is lost.

Reply:
I thought about this, I think returning the Price object makes this function not useful directly. Also if it's just `return Price.fromstring(extract_text(node))` I'm not sure if it should exist at all (unless we plan to expand it in the future). What do you think?

Reply:
I'm fine with only having this function in zyte-common-items processors, not in zyte-parsers. But if it's in zyte-parsers, returning Price looks more useful, even if the implementation is simple. It follows the idea "give me a selector, zyte-parsers extract an attribute".